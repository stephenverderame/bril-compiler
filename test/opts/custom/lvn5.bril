# ARGS: 4.33
@main(y: float) {
    x: int = const 10;
    print x;
    x2: float = const 10;
    z: float = fmul x2 y;
    print z;
    x3: float = id x2;
    x4: float = id x3;
    print x4;

}