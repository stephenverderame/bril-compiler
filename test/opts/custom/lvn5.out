10
43.29999999999999716
10.00000000000000000
